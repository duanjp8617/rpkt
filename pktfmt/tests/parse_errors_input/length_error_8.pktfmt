packet FakeProto {
    header = [
        field_1 = Field {
            bit = 8, 
        },
        field_2 = Field {
            bit = 8, 
            gen = false,
        },
        field_3 = Field {
            bit = 8, 
        }
    ],
    length = [
        header_len =field_5*3,
        payload_len =field_2*3,
    ]
}

