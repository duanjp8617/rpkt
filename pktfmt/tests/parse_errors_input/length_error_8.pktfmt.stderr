ast error
at /home/djp/pkt-rs/pktfmt/tests/parse_errors_input/length_error_8.pktfmt 14:5-17:5
         ~~~~~~~~~~
14 |     length = [
15 |         header_len =field_5*3,
16 |         payload_len =field_2*3,
17 |     ]
     ^^^^^
length error 8:
invalid length expression field name field_5
